### PR TITLE
adds unique discriptions to the grey sec uniforms

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
add: Grey security uniforms have unique names and descriptions
/:cl:
Partially Fixes: #24589 

Every piece of clothing with a unique icon needs a unique name. DON'T BE LAZY

### CODE REVIEW

File: code/modules/clothing/under/jobs/security.dm [hunk 1]
```diff
@@ -40,6 +42,8 @@
 	random_sensor = 0
 
 /obj/item/clothing/under/rank/warden/grey
+	name = "grey security suit"
+	desc = "A formal relic of years past before nanotrasen decided it was cheaper to dye the suits red instead of washing out the blood."
```
Comment (coiax) [inline code/modules/clothing/under/jobs/security.dm]: *Nanotrasen

File: code/modules/clothing/under/jobs/security.dm [hunk 2]
```diff
@@ -22,6 +22,8 @@
 	random_sensor = 0
 
 /obj/item/clothing/under/rank/security/grey
+	name = "grey security jumpsuit"
+	desc = "A tactical relic of years past before nanotrasen decided it was cheaper to dye the suits red instead of washing out the blood."
```
Comment (coiax) [inline code/modules/clothing/under/jobs/security.dm]: *Nanotrasen

**Comment by Kromgar** [comment]
But blood turns brown after drying THIS ISNT REALISTIC AT ALL

**Comment by Jazaen** [comment]
Human blood does that. Here, we are talking about lizards and spessmen.

**Comment by Supermichael777** [comment]
Centrals decision making shall not be questioned.

File: code/modules/clothing/under/jobs/security.dm (hunk 1 above)
Comment (Thunder12345) [inline code/modules/clothing/under/jobs/security.dm]: **NanoTrasen

File: code/modules/clothing/under/jobs/security.dm (hunk 2 above)
Comment (Thunder12345) [inline code/modules/clothing/under/jobs/security.dm]: **NanoTrasen

**Pull request merged**